src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/partial/secondFactor.html.twig

```diff
@@ -13,7 +13,16 @@
         </p>
         <hr>
         <p>
-            {{ ('ss.registration.selector.'~type~'.description')|trans }}
+            {{
+                ('ss.registration.selector.'~type~'.description')|trans(
+                    {
+                        '%tiqr_android_link_start%': '<a href="'~tiqrAppAndroidUrl~'">',
```

Comment:
They are escaped, see ```|raw``` on: https://github.com/SURFnet/Stepup-SelfService/pull/98/files#diff-43e7ab398b2e3d7574c3dd5c0f568be9R24

There appears to be no way around ```raw``` 😢